server/wages/views.py

```diff
@@ -72,21 +81,32 @@ def get_wages(request):
     return JsonResponse(_get_wages_serialized(request.GET))
 
 
-def _get_wages_serialized(query_params):
+def _get_query_size_limit_to_use(query_params):
```

Comment:
Might be sufficient to name the function _get_query_size_limit since it is assumed you will be using it.